Labs/Lab1 - Partner Center Setup/README.md

```diff
@@ -1,25 +1,25 @@
 
 # Lab 1: Partner Center Setup for a Virtual Machine Offer
 
-This lab will walk you through setting up a Virtual Machine Offer in Partner Center. You will need to ensure that you have created an account on Partner Center and have enrolled into the Marketplace Offers Program in order to be able to create Offers for the Marketplace. 
+This lab will walk you through setting up a Virtual Machine Offer in Partner Center. You will need to ensure that you have created an account in Partner Center and have enrolled into the commercial marketplace program in order to be able to create Offers in the Marketplace. 
 
 <br />
 
 > Note: This lab is part of a series of labs for Mastering the Virtual Machine Offers Workshop. You should finish this lab before moving on to the next sections. 
 
 <br />
 
-### Requisites before starting this lab
-1. Partner Center Account with enrollment in Marketplace Offers
-2. Access to the Azure Portal on the same tenant as is on Partner Center (Same tenancy is required to ensure you can publish images out of Compute Galleries, more on this in the later modules).
+### Prerequisites before starting this lab
+1. Partner Center Account with enrollment in the commercial marketplace program
```

Comment:
When I'm working with workspaces enabled, the section is called Marketplace offers. Maybe we change it to commercial marketplace program/marketplace offers so that if someone is working with workspaces disabled, then they know what to see